mozilla_connect.py

```diff
@@ -349,21 +377,24 @@ def main():
 
 	with open(os.path.join(adir, "Mozilla Connect_kudos.csv"), "w", newline="", encoding="utf-8") as csvfile:
 		writer = csv.writer(csvfile)
-		writer.writerow(("Kudos", "Board", "Labels", "Idea Status", "Subject", "Body", "URL"))
+		writer.writerow(("Kudos", "Total Kudos", "Date", "Board", "Labels", "Idea Status", "Subject", "Body", "URL"))
```

Comment:
Could mention the timezone used in the Date column's label?

Reply:
No problem, I updated it to convert the Connect dates to UTC. I also updated the headers for Connect, BMO and GitHub to specify the timezone. Please let me know if you want any additional changes.